waku.md

```diff
@@ -227,12 +228,12 @@ payload         = *OCTET
 padding         = *OCTET
 
 ; 65 bytes, if present.
-signature       = 65*OCTET
+signature       = *65OCTET
 
 ; 2 bytes, if present (in case of symmetric encryption).
-salt            = 2*OCTET
+salt            = 2OCTET
 
-envelope        = flags auxiliary-field payload padding [signature] salt
+envelope        = flags auxiliary-field payload padding [signature] [salt]
```

Comment:
two optionals here may seem fishy, but it makes sense with additional constraints (flag indicated) - perhaps we can pull this out into additional constraints section to keep the abnf more tight?